### ISSUE TITLE
The link issue in http://pulsar.apache.org/en/contact/

### ISSUE THREAD

**Issue by jiazhai** [apache/pulsar#2860]
In [contact](http://pulsar.apache.org/en/contact) page, the links has some issue: 
```
    const mailingLists = [
      {
        email: 'users@pulsar.apache.org',
        desc: 'User-related discussions',
        subscribe: 'mailto:users@pulsar.apache.org',     < === users-subscribe@..
        unsubscribe: 'mailto:users-unsubscribe@pulsar.apache.org',
        archives: 'http://mail-archives.apache.org/mod_mbox/pulsar-users/'
      },
      {
        email: 'dev@pulsar.apache.org',
        desc: 'Development-related discussions',
        subscribe: 'mailto:dev@pulsar.apache.org',    < ===  users-subscribe@..
        unsubscribe: 'mailto:dev-unsubscribe@pulsar.apache.org',
        archives: 'http://mail-archives.apache.org/mod_mbox/pulsar-dev/'
      },
      {
        email: 'dev@pulsar.apache.org',  < === commits@..
        desc: 'All commits to the Pulsar repository',
        subscribe: 'mailto:commits-subscribe@pulsar.apache.org',
        unsubscribe: 'mailto:commits-unsubscribe@pulsar.apache.org',
        archives: 'http://mail-archives.apache.org/mod_mbox/pulsar-commits/'
      }
    ]
```
Thanks for the guy that report this,

**Comment by quiye** [comment]
Just now I've found it too. I'll fix it.
https://github.com/apache/pulsar/blob/96c10dc5d2d55bb9132f25b465e9062855e42285/site2/website/pages/en/contact.js#L17-L39

**Comment by merlimat** [comment]
fixed in #2861
